<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Master</class>
 <widget class="QMainWindow" name="Master">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="isBroatcast">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>broadcast</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>command</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>send</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>recive</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timeout [ms]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeout">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Retrasmition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time limit [ms]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timelimit">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="message">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hex">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MASTER</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="address">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>247</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>address:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="receivedMessage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>received message</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
